modules/packages/TOML.chpl

```diff
@@ -21,6 +21,21 @@
 Chapel's Library for `Tom's Obvious, Minimal Language (TOML)
 <https://github.com/toml-lang/toml>`_.
 This module provides support for parsing and writing toml files.
+
+
+  .. note::
+
+    The TOML library for chapel is a work in progress. Many of the types listed in the TOML spec are not supported by this library. A core group of the spec is followed, but the library currently does not support the following types:
```

Comment:
nitpick: spec -> [TOML specification](https://github.com/toml-lang/toml)

Hyperlink would be useful too.